examples/with-webpack-bundle-size-analyzer/pages/about.js

```diff
@@ -1 +1,30 @@
-export default () => <div>About us</div>
+import Link from 'next/link'
+
+const About = ({ name }) => {
+  return (
+    <div>
+      <h1>About Page</h1>
+      <p>Welcome, {name}.</p>
+      <p>
+        This page is using getStaticProps, so the name will always be {name}{' '}
+        even if you reload the page.
+      </p>
+      <div>
+        <Link href="/">
+          <a>Home Page</a>
+        </Link>
+      </div>
+    </div>
+  )
+}
+
+export async function getStaticProps() {
+  // Runs only in the client
+  return {
+    props: {
+      name: 'Arunoda',
```

Comment:
Use `faker` here too, the name will always be the same even though `faker` generates a random name, because we only run `getStaticProps` once.